### ISSUE TITLE
Move architecture discription from introduction to a separate concept page

### ISSUE THREAD

**Issue by yveshanoulle** [openhab/openhab-docs#838]
I went to to introduction page, trying to read it with the view of a new user who has no programming background. I noticed the architecture at the bottem, while it explains a lot to people who have a programming background, I think it confuses people who don't.
Adding that on it a separate page unders concept would be much better + easier to find it back for people looking for it, on a later phase.

**Comment by MyRaceData** [comment]
I also have been looking at the introduction page with an eye toward how it may be perceived by a new user and agree the architecture overview is confusing to a new user. While some new users who are technically astute or have, as mentioned, a programming background may find it informative or interesting, most new users may find the section confusing. I also worry that some new users may feel understanding this section is required to use openHAB and it may scare them off before they even try to use the software.

**Comment by yveshanoulle** [comment]
I agree that more advanced users will like it, yet when it's under concepts, with a title architecture, tehy will see it and can look at it.
More over if it's there, they find it easier back later (I would never look at the first intro page to look for the archicture.)

**Comment by MyRaceData** [comment]
I also consider the introduction section to be not the best place for the architecture section.  Your suggestion to move it to the Concept section on its own page is, I believe, perhaps a better location. Especially if it is on its own page and furthermore I would suggest links to such a page should be marked as further reading/information for interested parties but not necessary for using openHAB.
My first thought on reading the introduction page recently was to delete the entire architecture section completely as I remember having no clue what it meant and made the software seem more complex then it turned out to be when I was a new user. Now your idea of moving seems better as some users may find this information interesting. Perhaps an page with title 'How it works' could include this section with a note at the top stating this information isn't necessary to run openHAB and just included for those who are interested.

**Comment by Confectrician** [comment]
If i am right the concepts section gets copied over from eclipse smarthome.
(Not sure at all for all of the pages, but it seems that the whole section is copied over)
So we would have to find out whats copied over and how we can merge this with openHAB only documentation. (Introduction is maintained in this repo.)

**Comment by LightIsLife** [comment]
hought you may want to know about my proposed revisions of the whole Intro section, addressing also this issue: https://community.openhab.org/t/introduction-in-docs-proposed-rewrite/62772

**Comment by MyRaceData** [comment]
Thanks for the heads up and some great feed back on the forum from everyone on the proposed rewrite. In your proposed rewrite is the architecture section removed? (I think it is) Do we need to find permanent new home for architecture section? If all of this has been considered and taken care of then this PR could probably be closed but I am not the author of the PR and so therefore then would defer to Yves

**Comment by LightIsLife** [comment]
Currently, this architecture section is still part of the proposed
rewritten doc. I just changed the heading.
But I am happy to move it wherever there is a better place. I am aiming at
putting a PR request tomorrow. Will mention this issue here and we can
discuss in the PR how to move this section (and the two news one I am
proposing) to the Concept section, which I understand is in the ESH repo.
So maybe we need a different place if this is becoming too complicated.

**Comment by MyRaceData** [comment]
Thanks for reply Markus. I agree with Yves that the introduction is not the best place for the architecture section but I'm glad you changed the heading and I suggest we move forward with the rewrite of the Introduction section PR and worry about moving the architecture section at a later date in the name of moving the process forward, Perhaps when the Concepts section receives the same treatment would be good time or Yves can make suggestion and bring this PR to fruition. Great work on the re-write !!!

**Comment by yveshanoulle** [comment]
somehow I did something wrong and asked github to not tell me about updates here :-( 

anyway, I'm back.

My take is, you first publish your rewrite (I'm a fan of finishing stuff before starting something else)
I'll keep the issue open, as a reminder.

Concepts was a place it felt more correct, yet if that's not in this repo, then I understand we need another place.
Maybe a section on it's own? I assume architecture can bundle a lot of things. that we don't have place for now.

**Comment by Confectrician** [comment]
Take a look @ https://www.openhab.org/docs/ @yveshanoulle 

I think the concepts page is not really misplaced in the current version of the article.
There is more than enough information regarding introdutional topics and it is clearly stated as a topic for advanced users.

**Comment by yveshanoulle** [comment]
It's better, u

> Take a look @ https://www.openhab.org/docs/ @yveshanoulle
> 
> I think the concepts page is not really misplaced in the current version of the article.
> There is more than enough information regarding introdutional topics and it is clearly stated as a topic for advanced users.
> 
> THis way technical informed users have the ability to get more technical information while in forming about openHAB in general.

It's better. yet a deeper dive in architecture does not belong in an introduction if you ask me.
